moda-contracts/src/interfaces/Releases/IReleasesOpen.sol

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.21;
+
+/**
+ * @title IReleases
+ * @dev Interface for Releases
+ */
+interface IReleasesOpen {
+    event OpenReleaseCreated(uint256 tokenId, address indexed releaseOwner);
+    event OpenReleaseRemoved(uint256 tokenId, address indexed releaseOwner);
+
+    /**
+     * @dev Creates a new release.
+     * @notice The msg.sender must be the owner of the tracks
+     * @param royaltyAmount - The percentage of sale prices
+     * that should be paid to the beneficiary for re-sales.
+     *  Calculated by <NOMINATOR> / 10,000. e.g. For 10% royalties, pass in 1000
+     * @param uri_ - The URI of the token metadata
+     * @param trackIds - An array containing the registered track ids of the tracks
+     */
+    function create(
+        uint96 royaltyAmount,
+        string calldata uri_,
+        uint256 amount,
+        string[] calldata trackIds
+    ) external;
+
+    /**
+     * @dev Returns info about the Releases contract including the total number of releases
+     * and where or not the contract is open or curated
+     * @return numberOfReleases - The total number of releases
+     * @return isOpen - Whether or not the contract is open or curated
+     */
+    function info() external returns (uint256 numberOfReleases, bool isOpen);
+
+    /**
+     * @dev Sets the metadata URI for a release
+     * @param tokenId - The ID of the token
+     * @param uri_ - The URI for the token
+     */
+    function setUri(uint256 tokenId, string calldata uri_) external;
+
+    /**
+     * @dev Removes a release
+     * @param tokenId - The ID of the token
+     */
+    function removeRelease(uint256 tokenId) external;
```

Comment:
- Let's just call this `burn`
- remove all other functions in this file
- rename file to `IBurnable`

Then `OpenReleases` is `IReleases, IBurnable`